src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <>
+      <section className="dp-gray-page p-t-54 p-b-54">
+        <div className="dp-rowflex p-t-96 p-l-96 p-r-96">
+          <div className="col-lg-6">
```

Comment:
You should wrap this between, something like this: 
```
<div className="dp-container">
   <div className="dp-rowflex">
       <div className="col-lg-6">.....</div>
       <div className="col-lg-6">.....</div>
   </div>
</div>
```

Reply:
change applied

Reply:
Could you update the screenshot in PR description, please? I think that with this change it should look different.

Reply:
change done  !